src/core/Akka.Remote/AkkaProtocolSettings.cs

```diff
@@ -51,11 +51,9 @@ public AkkaProtocolSettings(Config config)
             // backwards compatibility with the existing dot-netty.tcp.connection-timeout
             var enabledTransports = config.GetStringList("akka.remote.enabled-transports", new string[] { });
             if (enabledTransports.Contains("akka.remote.dot-netty.tcp"))
-                HandshakeTimeout = config.GetTimeSpan("akka.remote.dot-netty.tcp.connection-timeout", null);
-            else if (enabledTransports.Contains("akka.remote.dot-netty.ssl"))
-                HandshakeTimeout = config.GetTimeSpan("akka.remote.dot-netty.ssl.connection-timeout", null);
```

Comment:
Removed `akka.remote.dot-netty.ssl`, this HOCON path has not been used since 5 years ago

Reply:
LGTM